charts/descheduler/README.md

```diff
@@ -84,6 +84,7 @@ The following table lists the configurable parameters of the _descheduler_ chart
 | `serviceMonitor.metricRelabelings`  | MetricRelabelConfigs to apply to samples after scraping, but before ingestion                                         | `[]`                                      |
 | `serviceMonitor.relabelings`        | RelabelConfigs to apply to samples before scraping                                                                    | `[]`                                      |
 | `affinity`                          | Node affinity to run the descheduler cronjob/deployment on specific nodes                                             | `nil`                                     |
+| `topologySpreadConstraints`         | Topology Spread Constraints to spread the descheduler cronjob/deployment across the cluster                           | `[]`                                     |
```

Comment:
Spread and skew does not apply to a Job as it's single replica (so there's never a skew)
```suggestion
| `topologySpreadConstraints`         | Topology Spread Constraints to spread the descheduler HA Deployment across the cluster                           | `[]`                                     |
```